### PR TITLE
[Application] define YOLO v2 model class

### PR DESCRIPTION
**Pull request by baek2sm**
define YOLO v2 model structure corresponding to pytorch example.

now, it is implemented by fake data, and there is no loss function for this model.

**Self evaluation:**
1. Build test: [x]Passed []Failed []Skipped
2. Run test: [x]Passed []Failed []Skipped

Signed-off-by: Seungbaek Hong <sb92.hong@samsung.com>

### CODE REVIEW

**Review by DonghakPark — COMMENTED** [review]
We usually make ```jni``` dir
How about move ```main.cpp``` and ```meson.build``` to ```Applications/YOLO/jni/```

**Comment by baek2sm** [comment]
> We usually make `jni` dir How about move `main.cpp` and `meson.build` to `Applications/YOLO/jni/`

I'll move it to jni dir. thanks!

**Comment by baek2sm** [comment]
> We usually make `jni` dir How about move `main.cpp` and `meson.build` to `Applications/YOLO/jni/`

It isn't setup for android yet, so I'll update it later!

**Review by jijoongmoon — APPROVED** [review]
LGTM!

File: Applications/meson.build [hunk 1]
```diff
@@ -9,6 +9,7 @@ if enable_ccapi
 endif
 subdir('VGG/jni')
 subdir('Resnet/jni')
+subdir('YOLO')
```
Comment (jijoongmoon) [inline Applications/meson.build:12]: may be add for the android?

Comment (baek2sm) [reply Applications/meson.build]: I added jni folder and related files. thanks!

**Review by DonghakPark — APPROVED** [review]
LGTM!
